tests/test_ipa_tracking.py

```diff
@@ -128,3 +132,165 @@ def test_one_stuck(self):
         result = self.results.results[0]
         assert result.result == constants.WARNING
         assert result.kw.get('key') == '7777'
+
+
+class TestFIPSTokens(BaseTest):
+    """Test the combination of FIPS configurations with tokens and
+       also that when HSM is used the check is skipped.
+    """
+    patches = {
+        'ipahealthcheck.ipa.certs.get_expected_requests':
+        Mock(return_value=get_expected_requests()),
+        'ipalib.install.certmonger._cm_dbus_object':
+        Mock(side_effect=create_mock_dbus),
+        'ipalib.install.certmonger._certmonger':
+        Mock(return_value=_certmonger())
+    }
+
+    nss_tracking = {
+        'nickname': '9876',
+        'ca-name': 'dogtag-ipa-ca-renew-agent',
+        'template_profile': 'caIPAserviceCert',
+        'cert-storage': 'NSSDB',
+        'cert-storage_location': '/etc/pki/pki-tomcat/alias',
+        'cert-token': 'NSS Certificate DB',
+        'key-storage': 'NSSDB',
+        'key-storage_location': '/etc/pki/pki-tomcat/alias',
+        'key-token': 'NSS Certificate DB',
+    }
+
+    @patch('ipaplatform.tasks.tasks.is_fips_enabled')
+    @patch('ipalib.install.certmonger.get_request_value')
+    def test_nonfips_token_correct(self, mock_value, mock_fips):
+        mock_value.side_effect = [
+            'FILE',
+            'FILE',
+            'NSSDB', 'NSS Certificate DB',
+        ]
+        mock_fips.return_value = False
+        set_requests(add=self.nss_tracking)
+#        mock_requests.return_value=get_expected_requests()
+#        mock_obj.side_effect=create_mock_dbus
```

Comment:
Can we tidy this up, or leave a comment to explain why these lines are commented out?

Reply:
Still got this (apparently?) vestigal commented-out code.  Not a blocker.  Leave it to you whether to remove or not.  (Maybe it's there for a reason...)